### PR TITLE
Upgrade SBT to 1.5.6 and Play to 2.8 boom

### PR DESCRIPTION
**Pull request by nicl**
## What does this change?

Bump SBT version to 1.5.6 and related changes.

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -1,15 +1,12 @@
-resolvers += "Typesafe Releases" at "https://repo.typesafe.com/typesafe/releases/"
```
Comment (jacobwinch) [inline project/plugins.sbt:1]: Would you mind explaining why this resolver is no longer required? (Just for my own understanding!)

File: project/plugins.sbt [hunk 2]
```diff
@@ -1,15 +1,12 @@
...
-
 // keep in sync with the play version in Dependencies
```
Comment (jacobwinch) [inline project/plugins.sbt:1]: Do you think this comment is still relevant? 

I noticed that the Play version here does not match the version of `play-json` in Dependencies (this was also true before this PR!)

**Review by jacobwinch — APPROVED** [review]
Great work 👍

**Comment by nicl** [comment]
@jacobwinch sorry yeah, meant to add WIP. I'm going to close this and split into some pre-PRs (to e.g. remove Guardian Management) to make things easier to review and less gigantic.

**Pull request closed without merging**